docs/content/direct/knownissue-wsl-ghcr-helm.md

```diff
@@ -1,20 +1,62 @@
-# Authorization failure in WSL while fetching Helm chart from ghcr.io
+# Authorization failure in WSL/Ubuntu while fetching Helm chart from ghcr.io
 
 ## Description of the Issue
 
-When using WSL (Windows Subsystem for Linxu) and following the
-[Getting Started recipe](get-started.md) you might get a failure from
-the command to instantiate KubeStellar's core Helm chart. The error
-message is as follows.
+When working with WSL (Windows Subsystem for Linux)/Ubuntu and trying to instantiate KubeStellar's core Helm chart, you might encounter the following error:
 
-> Error: failed to authorize: failed to fetch oauth token: unexpected status from GET request to https://ghcr.io/token?scope=repository%3Akubestellar%2Fkubestellar%2Fcore-chart%3Apull&service=ghcr.io: 403 Fobidden
+> Error: failed to authorize: failed to fetch oauth token: unexpected status from GET request to https://ghcr.io/token?scope=repository%3Akubestellar%2Fkubestellar%2Fcore-chart%3Apull&service=ghcr.io: 403 Forbidden
 
-This is [Issue 2544](https://github.com/kubestellar/kubestellar/issues/2544).
+This error indicates that there was an issue fetching the OAuth token from GitHub's container registry (`ghcr.io`), which is preventing you from pulling the Helm chart.
 
 ## Root Cause
 
-Unknown.
-
+The issue is caused by missing or misconfigured credentials for `ghcr.io` in the Docker configuration. Docker fails to authenticate & retrieve the necessary OAuth token from GitHub's container registry.
 ## Workaround
 
-Run the Getting Started recipe as root user in Linux. For example, use `sudo su -` to get a shell as root.
+To bypass this issue, follow these steps:
+
+1. Run the Getting Started recipe as the root user in Linux. This can be done using the `sudo su -` command to get a shell as root.
+
+```bash
+sudo su -
+```
+
+This will allow you to execute the Helm chart instantiation command with the necessary privileges.
+
+## 2. Authenticate Docker with GitHub Container Registry
+
+Docker requires authentication to access `ghcr.io` (GitHub's container registry). To authenticate, you need to log into the registry using your GitHub token.
```

Comment:
Not exactly. The chart is public, no login is needed. Your problem was that your Docker Desktop installation was broken, due to not installing `pass`. The Docker Desktop installation instructions say that you should install `pass`. Without it, Docker Desktop just does not work.